tests/tr1/tests/random5/test.cpp

```diff
@@ -486,9 +499,10 @@ static void tpiecewise_constant() {
     dist_t dist2(10, 1.0, 2.0, myfn);
     CHECK_INT(dist2.densities().size(), 10);
 
-    double arr[] = {1.0, 1.1, 1.2, 1.3, 1.4};
+    double arr[] = {1.0, 1.5, 2.0, 3.0, 4.0};
     dist_t dist3(STD initializer_list<double>(&arr[0], &arr[5]), myfn);

```

Comment:
Ugh, using unrelated library internals in a test for no reason at all: tr1 things. (No change requested.)

![image](https://user-images.githubusercontent.com/456873/89967345-147ef300-dc06-11ea-9243-3d938389bac6.png)

Reply:
Could be Good First Issue other than update define

Reply:
A widespread problem #1184